base/uk.ac.stfc.isis.ibex.ui.devicescreens/src/uk/ac/stfc/isis/ibex/ui/devicescreens/models/EditDeviceScreensDescriptionViewModel.java

```diff
@@ -324,16 +439,13 @@ public void deleteScreen(int index) {
     public void addScreen() {
         List<DeviceDescriptionWrapper> oldList = new ArrayList<>(devices);
 
-        DefaultName namer = new DefaultName("Screen", "_", false);
+        DefaultName namer = new DefaultName("Screen", "_", this.canWriteRemote);
```

Comment:
The third argument just specifies whether the allocated default name should contain brackets. You should just pass "false" as brackets aren't allowed in device screen names.